src/pages/kb/user-guide/visualizations/visualizations-how-to.md

```diff
@@ -26,12 +26,21 @@ It's easy to embed Redash visualizations. Just click the elipsis button beneath
 This will pop up the `<iframe>` code you can drop into your HTML pages.
 
 {% callout warning %}
-Queries with parameters do not support embeds at the moment.
+Queries with text-type parameters do not support embeds.
 {% endcallout %}
 
-For SaaS customers, there is also a hardlink to a PNG of your visualization hosted through `snap.redash.io`. The PNG embed is especially useful in contexts where iframes won't work (like GitHub issues). 
+For SaaS customers, there is also a hardlink to a PNG of your visualization hosted through `snap.redash.io`. The PNG embed is especially useful in contexts where iframes won't work (like GitHub issues).
+
+### Query String Variables
+
+You can append query string variables to your embed URLs:
+
+* `?no-cache` will add a `Cache-Control: no-cache` header to an embedded PNG.
```

Comment:
Putting this along side the others gives the impression it will work for the PNG embed, but it won't.

Reply:
Fixed.